include/bluetooth/audio.h

```diff
@@ -1172,10 +1172,82 @@ struct bt_audio_broadcast_sink_cb {
 
 /** @brief Stream operation. */
 struct bt_audio_stream_ops {
+	/** @brief Stream configured callback
+	 *
+	 *  Configured callback is called whenever an Audio Stream has been
+	 *  configured.
+	 *
+	 *  @param stream Stream object that has been configured.
+	 */
+	void (*configured)(struct bt_audio_stream *stream);
+
+	/** @brief Stream QoS set callback
```

Comment:
Wonder whether it would be too general to call it the QoS callback (without the "set").
But thinking about it, the "set" is nice to have, as it gives the same form of the name as the other callbacks.
So - keep as is.

Reply:
We could use the `_cb` suffix you have used for Media Control. It's inconsistent with more callbacks in the BT stack, but it does give some more meaning to the name of the callback.

Reply:
Nah - I was first considering proposing removing the "set", but I think it is better to keep it.
No need for _cb suffix, I think.